### PR TITLE
corrections and English clarified

### PR DESCRIPTION
**Pull request by mattsmi**
A couple more markdown files corrected. 
There are some good definitions in deck-options.md now. But a Mature Card is not defined. Should it be?

Second question: Spelling is mixed in some of the markdown files. Are you standardising on British (which includes Australian) or US English? I have assumed the former, given some of the older content, which I assume was authored by your fine self.

Thanks again, Damien!

### CODE REVIEW

File: src/deck-options.md [hunk 1]
```diff
@@ -390,26 +390,26 @@ If you plug the resulting 65% into the interval modifier, you should
 find over time that your retention moves closer to your desired
 retention.
 
-One important thing to note however is that the tradeoff between time
+One important thing to note however is that the trade-off between time
 spent studying and retention is not linear: we can see here that to
-increase our retention by 5 percentage points, we’d have to study 35%
+increase our retention by 5 per cent, we would have to study 35%
```
Comment (dae) [inline src/deck-options.md:395]: I'm not sure this change is correct?

https://thewritingbusiness.com/the-difference-between-percentages-and-percentage-points/

**Comment by dae** [comment]
Thanks Matt! Australian is my mother tongue, but using American English when programming and writing translations seems to be more common, and American-English speakers seem to be in the majority, so I tend towards using that these days. I don't have strong preferences for the manual - we're all volunteers here, from different backgrounds, and documentation in a mix of dialects is better than none at all. :-)

**Comment by dae** [comment]
Mature is documented here, but if you think it needs to be covered again, please add it by all means: https://docs.ankiweb.net/stats.html?highlight=mature#types-of-cards

File: src/deck-options.md (hunk 1 above)
Comment (mattsmi) [reply src/deck-options.md:395]: Yes, you are right! Sorry, I think the whiskey must have run out. :)
Will you revert this change, or should I do it and resubmit a pull request?
Many thanks for you time and sorry the error!

Comment (dae) [reply src/deck-options.md:395]:
```suggestion
increase our retention by 5 percentage points, we would have to study 35%
```

The interface should give you an option to accept this suggested change, and then I can merge the PR.

Comment (dae) [reply src/deck-options.md]: Nevermind, I can do that myself :-)

**Pull request merged**